apps/healthlocker/lib/oxleas_adhd/edit_room.ex

```diff
@@ -11,10 +11,31 @@ defmodule Healthlocker.OxleasAdhd.EditRoom do
     |> Multi.run(:clinician_room, &add_clinicians_to_room(&1, clinician_ids, room))
   end
 
-  def connect_teachers_and_update_rooms(room, teacher_ids, teachers, query) do
+  def connect_teachers_and_update_rooms(service_user, teacher_ids, teachers, query) do
     Multi.new
     |> Multi.delete_all(:delete_teachers, query)
     |> Multi.insert_all(:insert_teachers, Teacher, teachers)
+    |> Multi.insert_all(:teacher_rooms, Room, add_teachers_room(service_user.id, teacher_ids))
+  end
+
+  defp add_teachers_room(user_id, teacher_ids) do
+    existing_rooms = Repo.all(from r in Room, where: ilike(r.name, ^"teacher-care-team:#{user_id}%"))
```

Comment:
that's really cool. I haven't seen `ilike` before. Will def be using that going forward 👍